### ISSUE TITLE
The Evaluation on YCBV Dataset

### ISSUE THREAD

**Issue by zhujunli1993** [ethnhe/FFB6D#46]
Hi. 

I tried to test the given trained model on YCBV dataset, and I got the following results:
Average of all object:
***************add:  90.29947942321212
***************adds:     96.66162312565027
***************add(-s):  93.97500795337722
All object (following PoseCNN):
***************add:  90.538107975554
***************adds:     96.59589003702112
***************add(-s):  93.01384107439492

I'd like to know if 'adds' represents ADDS and 'add(-s)' represents ADD(S) in the paper? Thanks!!

**Comment by ethnhe** [comment]
Yes, the All object adds and add(-s) results are reported. After the MeanShift problem is fixed, we get better add(-s) than the paper-reported results.
